### PR TITLE
task/send_image/camera_fab

### PR DESCRIPTION
**Pull request by teamagam1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/19" target="_blank">ReviewNinja

File: app/src/main/res/layout/fragment_cesium.xml [hunk 1]
```diff
@@ -1,32 +1,52 @@
...
-        />
+        android:layout_height="match_parent" />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="right|bottom"
+        android:orientation="vertical">
+
+        <android.support.design.widget.FloatingActionButton
```
Comment (BarAgam) [inline app/src/main/res/layout/fragment_cesium.xml:21]: use style to decor the FABs and avoid styling dups !fix

Comment (teamagam1) [inline app/src/main/res/layout/fragment_cesium.xml:21]: !fixed

Comment (teamagam1) [inline app/src/main/res/layout/fragment_cesium.xml:21]: !fixed

**Pull request merged**